c2corg_api/tests/views/test_cooker.py

```diff
@@ -0,0 +1,13 @@
+from c2corg_api.tests.views import BaseTestRest
+
+
+class TestCookerRest(BaseTestRest):
+    def setUp(self):  # noqa
+        super(TestCookerRest, self).setUp()
+
+    def test_get(self):
+        body = {
+            "lang": "fr",
+            "oui": "**coucou** *italic*"
+        }
+        self.app_post_json('/cooker', body, status=200)
```

Comment:
Actually, it's not the job of this repo to check markdown. By the way, I can add two assertion : 

* lang value's must not be touched, as it's not a markdown field
* oui value's must be modified, because it's not part of NOT_MARKDOWN_PROPERTY

Reply:
True, in-depth tests of the Markdown parser are not required. I was more thinking of a very basic test to see that it works at all.